packages/block-editor/src/components/block-tools/style.scss

```diff
@@ -78,6 +78,7 @@
 		color: $white;
 		padding: 0;
 
+		// TODO: Consider passing size="small" to the Inserter toggle instead.
 		// Special dimensions for this button.
 		min-width: $button-size-small;
 		height: $button-size-small;
```

Comment:
Virtually all usage in Gutenberg seem to be overriding the toggle Button size to be 24px. They should consider using `size="small"` instead of doing this with custom CSS (out of scope for this PR, so I left TODO comments).